wiki/People/Beatmap_Nomination_Group/Rules/en.md

```diff
@@ -0,0 +1,57 @@
+# Beatmap Nominator Rules
+
+The rules listed here affect what Beatmap Nominators can and cannot do when nominating beatmaps as well as set the tone for the general conduct expected from them. 
```

Comment:
why though? This entire page lists both things nominators can and cannot do, some even within the same ruling